portfolio/src/main/webapp/script.js

```diff
@@ -12,37 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+var numComments = 1;
```

Comment:
Question: Is this global variable used to make sure when people refresh the get the same results?

Reply:
Ah no this is the default number of comments. If people do not have inputs for "number of comments", it will set the default number of comments to display as 1.

Reply:
I have changed the variable name and definition to make things clearer for readability